tests/integration/test_fim/test_registry/test_checks/test_check_all.py

```diff
@@ -0,0 +1,177 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
```

Comment:
All three tests on this file are pretty similar between eachother, could we maybe parametrize a couple more things like the tag to apply and the options used in the test and unify them?